tensorflow/core/kernels/BUILD

```diff
@@ -2128,7 +2128,10 @@ tf_kernel_library(
 tf_kernel_library(
     name = "matmul_op",
     defines = select({
```

Comment:
Is there any reason why we wouldn't want to always enable libxsmm for CPUs that support it?